src/unoserver/server.py

```diff
@@ -14,55 +14,51 @@ class UnoServer:
     def __init__(self, interface="127.0.0.1", port="2002"):
         self.interface = interface
         self.port = port
+        self.user_installation = None
```

Comment:
You should pass user_installation as an argument to __init__() as well.

Reply:
I'll move it back